snapshot/cli/cli.go

```diff
@@ -220,38 +234,42 @@ func (c *createGitBundleCommand) run(db snapshot.DB, _ *cobra.Command, _ []strin
 		return fmt.Errorf("not a valid repo dir: %v, %v", wd, err)
 	}
 
-	bundleFilename, tempDir := "", ""
-	revList := fmt.Sprintf("%s..%s", c.basis, c.ref)
-
-	if c.file != "" {
-		bundleFilename = c.file
-	} else {
-		td, err := temp.TempDirDefault()
-		if err != nil {
-			return fmt.Errorf("Couldn't create temp dir: %v", err)
-		}
-		tempDir = td.Dir
-
-		// Don't use commit sha as bundle name as it could collide with other bundles.
-		// Use the rev-list for the given basis/ref to generate a unique sha
-		revData, err := ingestRepo.Run("rev-list", revList)
-		if err != nil {
-			return fmt.Errorf("couldn't get rev-list for %s: %v", revList, err)
-		}
-		log.Infof("Using rev-list for %s:\n%s\n", revList, revData)
-		bundleFilename = path.Join(tempDir, fmt.Sprintf("bs-%x.bundle", sha1.Sum([]byte(revData))))
+	gdb, ok := db.(*gitdb.DB)
+	if !ok {
+		return fmt.Errorf("create bundle requires a gitdb.DB snapshot.DB")
 	}
 
+	td, err := temp.TempDirDefault()
```

Comment:
This is needed by the create bundle command. Prefer to leave it up with the other prereq stuff so that the business logic is more readable